### PR TITLE
[3rdparty] Doctrine dbal to 2.9.2

### PR DESCRIPTION
**Pull request by rullzer**
Requires;
- [x] PAssing CI
- [x] https://github.com/nextcloud/3rdparty/pull/283
- [x] rebase

### CODE REVIEW

**Comment by rullzer** [comment]
Whee seems good!

**Review by ChristophWurst — APPROVED** [review]
Yesssss :rocket:

**Comment by nickvergessen** [comment]
Weeeee 2.9 :-)

File: lib/private/DB/OCSqlitePlatform.php [hunk 1]
```diff
@@ -23,27 +23,5 @@
 namespace OC\DB;
 
 class OCSqlitePlatform extends \Doctrine\DBAL\Platforms\SqlitePlatform {
```
Comment (kesselb) [inline lib/private/DB/OCSqlitePlatform.php]: You could drop `OCSqlitePlatform` ;)

Comment (rullzer) [reply lib/private/DB/OCSqlitePlatform.php]: I know we could. But lets keep it for nwo. we have platform wrappers for everything else as well

Comment (nickvergessen) [reply lib/private/DB/OCSqlitePlatform.php]: Also this allows to easier see the history and where the content was removed etc.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**